spec/stat_tracker_spec.rb

```diff
@@ -7,13 +7,21 @@
 RSpec.describe Class do
 
   it "attributes" do
-    stattracker= StatTracker.new
-    expect(stattracker.games).to eq(nil)
-    expect(stattracker.teams).to eq(nil)
-    expect(stattracker.game_teams).to eq(nil)
+    game_path = './data/games.csv'
```

Comment:
I like how you removed the initialize test and included it in the from_csv test.